src/classes/CON_Preferred_TEST.cls

```diff
@@ -37,92 +37,116 @@
 @isTest
 public with sharing class CON_Preferred_TEST {
```

Comment:
Kudos for updating these tests to make sure we have tests that work for existing customers that have it disabled!!